Jenkinsfile

```diff
@@ -15,7 +15,7 @@ pipeline {
     }
     environment {
         RUST_IMAGE_REPO = 'us.gcr.io/logdna-k8s/rust'
-        RUST_IMAGE_TAG = 'buster-stable'
+        RUST_IMAGE_TAG = 'buster-1-stable'
```

Comment:
@c-nixon Without this, I was getting ``feature `resolver` is required`` from the Jenkins build when it was trying to download the time crate. I assume it is something to do with the version of cargo installed in the older image but not sure what the downstream impact of using the new build image would be. Is there another way to fix this?

Reply:
I think we are ok bumping the build image version, it shouldn't have any downstream impact as it isn't shipped or anything like that, it only generates the binaries